index.bs

```diff
@@ -351,10 +346,13 @@ spec: CSP; urlPrefix: https://www.w3.org/TR/CSP3/#
 
       1. If <var>origin</var> is an [=opaque origin=], return false.
 
+      1. Let <var>url</var> be the result of calling the [=url parser=] on the
+         [=serialization of an origin|serialization=] of <var>origin</var>.
+
       1. Otherwise, [=set/for each=] <a>permissions-source-expression</a> |item| in the <a>allowlist</a>'s <a>expressions</a>:
```

Comment:
LGTM -- just get rid of the "Otherwise" here, since the previous line isn't the "If" line anymore. (And that line returns anyway, so control flow would never get here if origin was opaque)